src/World.h

```diff
@@ -685,6 +685,9 @@ class cWorld :
 	/** Returns the number of chunks loaded	 */
 	int GetNumChunks() const;  // tolua_export
 
+	/** Returns the number of unused dirty chunks. That's the number of chunks that would be unloaded shortly after calling "SaveAllChunks" */
+	int GetNumUnusedDiryChunks(void);
```

Comment:
I exported. Do I need to manually modify `APIDesc`? Or have we got the automatic doxy import idea implemented?